spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/ldap/embedded/EmbeddedLdapAutoConfiguration.java

```diff
@@ -107,6 +109,28 @@ public InMemoryDirectoryServer directoryServer() throws LDAPException {
 					this.embeddedProperties.getCredential().getUsername(),
 					this.embeddedProperties.getCredential().getPassword());
 		}
+
+		Schema schema = null;
+		if (this.embeddedProperties.getValidation().isEnabled()) {
+			schema = Schema.getDefaultStandardSchema();
+			String schemaLocation = this.embeddedProperties.getValidation().getSchema();
+			if (StringUtils.hasText(schemaLocation)) {
```

Comment:
Rather than using a `String` there you could use a `Resource`. That way the binder will do the work of making sure that resource is valid (if set).